pkg/tfgen/test_data/resources/libvirt_network/docs.json

```diff
@@ -0,0 +1,75 @@
+  "Arguments": {
+    "addresses": {
+      "description": "A list of (0 or 1) IPv4 and (0 or 1) IPv6 subnets in\nCIDR notation.  This defines the subnets associated to that network.\nThis argument is also used to define the address on the real host.\nIf `dhcp {  enabled = true }` addresses is also used to define the address range served by\nthe DHCP server.\nNo DHCP server will be started if `addresses` is omitted."
+    },
+    "altering_libvirt.xslt": {
+      "description": ""
+    },
+    "autostart": {
+      "description": "Set to `true` to start the network on host boot up.\nIf not specified `false` is assumed."
+    },
+    "bridge": {
+      "description": "The bridge device defines the name of a bridge\ndevice which will be used to construct the virtual network (when not provided,\nit will be automatically obtained by libvirt in `none`, `nat`, `route` and `open` modes)."
+    },
+    "dns": {
+      "description": "configuration of DNS specific settings for the network"
+    },
+    "dns.dhcp": {
+      "description": "DHCP configuration. \nYou need to use it in conjuction with the adresses variable."
+    },
+    "dns.dnsmasq_options": {
+      "description": "configuration of Dnsmasq options for the network\nYou need to provide a list of option name and value pairs."
+    },
+    "dns.enabled": {
+      "description": "when false, disable the DHCP server\n```hcl\nresource \"libvirt_network\" \"test_net\" {\nname      = \"networktest\"\nmode      = \"nat\"\ndomain    = \"k8s.local\"\naddresses = [\"10.17.3.0/24\"]\ndhcp {\nenabled = true\n}\n```"
+    },
+    "dns.forwarders": {
+      "description": "Either `address`, `domain`, or both must be set"
+    },
+    "dns.hosts": {
+      "description": "a DNS host entry block. You can have one or more of these\nblocks in your DNS definition. You must specify both `ip` and `hostname`.\n\nAn advanced example of round-robin DNS (using DNS host templates) follows:\n\n```hcl\nresource \"libvirt_network\" \"my_network\" {\n...\ndns {\nhosts { flatten(data.libvirt_network_dns_host_template.hosts.*.rendered) }\n}\n...\n}\n\ndata \"libvirt_network_dns_host_template\" \"hosts\" {\ncount    = var.host_count\nip       = var.host_ips[count.index]\nhostname = \"my_host\"\n}\n```\n\nAn advanced example of setting up multiple SRV records using DNS SRV templates is:\n\n```hcl\ndata \"libvirt_network_dns_srv_template\" \"etcd_cluster\" {\ncount    = var.etcd_count\nservice  = \"etcd-server\"\nprotocol = \"tcp\"\ndomain   = var.discovery_domain\ntarget   = \"${var.cluster_name}-etcd-${count.index}.${discovery_domain}\"\n}\n\nresource \"libvirt_network\" \"k8snet\" {\n...\ndns {\nsrvs = [ flatten(data.libvirt_network_dns_srv_template.etcd_cluster.*.rendered) ]\n}\n...\n}\n```"
```

Comment:
These are `hcl` blocks so this is before example rendering? Trying to wrap my head around sequencing.

Reply:
Yes. We have a parse phase and then a render phase, with intermediate data represented in `entityDocs`. This change only effects the parse: raw markdown (`[]byte`) to `entityDocs`.